cli/cmd/cmdutil/cmdutil.go

```diff
@@ -169,6 +169,26 @@ func OrgExists(ctx context.Context, c *client.Client, name string) (bool, error)
 	return true, nil
 }
 
+func ProjectNames(ctx context.Context, c *client.Client, orgName, githubURL string) ([]string, error) {
```

Comment:
Only used once, and not particularly generic – move to internal function in the file that uses it for now